src/chainparams.cpp

```diff
@@ -198,6 +199,7 @@ class CTestNetParams : public CMainParams {
         vAlertPubKey = ParseHex("04302390343f91cc401d56d68b123028bf52e5fca1939df127f63c6467cdf9c8e2c14b61104cf817d0b780da337893ecc4aaff1309e536162dabbdb45200ca2b0a");
         nDefaultPort = 18333;
         nMinerThreads = 0;
+        nPruneAfterHeight = 1000; //! low since testnet can be reset in future
```

Comment:
If testnet is reset, it will just be a separate network (testnet4?), so that shouldn't be a concern.